jest.config.js

```diff
@@ -24,7 +24,12 @@ module.exports = {
 
   testEnvironment: 'node',
 
+  "transformIgnorePatterns": [
```

Comment:
this dumb configuration is required to make assemblyscript/loader work with both jest and ts-node
https://github.com/AssemblyScript/assemblyscript/issues/2139#issuecomment-1046250511